includes/class-wc-payments-checkout.php

```diff
@@ -330,8 +330,8 @@ public function get_enabled_payment_method_config() {
 				/* translators: link to Stripe testing page */
 				$payment_method->get_testing_instructions(),
 				[
-					'strong' => '<strong>',
-					'a'      => '<a href="https://woocommerce.com/document/woopayments/testing-and-troubleshooting/testing/#test-cards" target="_blank">',
+					'span' => '<span class="copy-icon" onclick="navigator.clipboard.writeText(`4242424242424242`)">',
```

Comment:
Unfortunately, this solution doesn't seem to be very accessible.

I'd recommend using a `button` element:
- It has an outline for keyboard focus
- You can navigate it with keyboard, and it is accessible by screen readers
- It has an active style to provide feedback when clicked

I'd also recommend adding an `aria-label` and a `title` attributes - something like "Copy the test card number to your clipboard" and "Copy to your clipboard", respectively (as an example)..

Reply:
Cheers for this insight and instruction. Fixed (hopefully) in 0148c12324ee2bda9563761a02876e862568c2c2.